### ISSUE TITLE
Adding voice search 

### ISSUE THREAD

**Issue by Praveen0899** [TeamNewPipe/NewPipe#1913]
Why can't we add the voice search in search box.

**Comment by hueldoeu** [comment]
Yes that would be nice. Then newpipe is truely perfect. I ithink its easy to use the google-in-app-voice-search, which is extreamly preciesly now

**Comment by Zeik0s** [comment]
Because building this feature requires some proprietary Google libraries, which can't be used on devices without play services. However, this app is built for such devices. Newpipe also advertises that it's independent from Google Libraries.

**Comment by Primokorn** [comment]
@theScrabi That would be very appreciated on Android TV since you support this platform.
NewPipe is my default YouTube client now but it's a pain to search with the keyboard on TV.
**Thanks anyway**

**Comment by TobiGr** [comment]
Are there any alternative voice search applications installed on your device?

**Comment by Primokorn** [comment]
Google voice. I have a Mi Box S device.
